baselines/dom.generated.d.ts

```diff
@@ -5044,6 +5049,7 @@ interface Element extends Node, ARIAMixin, Animatable, ChildNode, InnerHTML, Non
     readonly tagName: string;
     /** Creates a shadow root for element and returns it. */
     attachShadow(init: ShadowRootInit): ShadowRoot;
+    checkVisibility(options?: CheckVisibilityOptions): boolean;
```

Comment:
Just as a reference as there is not much content online:

https://github.com/mdn/browser-compat-data/blob/d22e1912155eb29056d254213b1fddbd7fc05391/api/Element.json#L2514

https://drafts.csswg.org/cssom-view/#dom-element-checkvisibility